docs/Developers/Newcomers_Guide.rst

```diff
@@ -3,74 +3,83 @@
-  - run coala on a project of yours
-  - merge a ``difficulty/newcomer`` Pull Request
-  - review at least a ``difficulty/newcomer`` Pull Request
-  - merge a ``difficulty/low`` Pull Request
-  - review at least a ``difficulty/low`` or higher Pull Request
-
-When you ran coala on a project, please fill our
-`usability survey <http://coala.io/usability>`_. Once you got your first Pull
-Request merged successfully, fill in our
-`survey form <http://coala.io/newform>`_. With that you can help us making your
-experience better!
-
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
-to name yourself a coala developer!
+This is a step by step guide that teaches you the workflow used at coala and
+how to use the tools involved with it.
+Although we did our best to provide as much help as possible, we are always
+there to help at http://coala.io/chat if you feel lost.
+**But please do read the guide before asking us as most questions we get are
+covered in here.**
+
+Before you can start working there are a few things you need to know about
+coala:
+
+  - Everyone in the coala community is expected to follow our
+    `Code of Conduct <http://coala.io/coc>`_.
+  - Do not work on any issue without assignment. Someone else might be working
+    on it already.
+  - When something is unclear try to find a solution yourself. This is a
+    valuable skill!
+  - Only after you tried to solve a problem yourself ask the
+    `chat <coala.io/chat>`_
+
+To become a coala developer, there's a few steps you need to complete.
+The newcomer process is as it follows:
+
+  - run coala on a project of yours and fill our
+    `usability survey <http://coala.io/usability>`_.
+  - merge a ``difficulty/newcomer`` Pull Request and fill in our
+    `survey form <http://coala.io/newform>`_.
+  - review at least a ``difficulty/newcomer`` Pull Request.
+  - merge a ``difficulty/low`` Pull Request.
+  - review at least a ``difficulty/low`` or higher Pull Request.
 
 .. note::
 
-    **Do not only fix a newcomer issue!** Supervising newcomers is really a lot
-    of work. We're all volunteers and we can't keep this up if you don't help
-    us in other areas as well!
+    Just giving an ack to a PR does not count as a review. Even if that PR is
+    in a mergable state. We want you to learn and develop an eye for finding
+    improvements. When in doubt, don't give an ack and keep reviewing.
 
-Of course, the order is not important, although, we recommend you to start
-with a ``newcomer`` issue, end with a ``low`` issue, and review other PRs in
-the meantime!
-
-This is a step-based guide that will help you get your first contribution
-at coala, making you familiar with the work flow!
+Once you have achieved all these, just ask for being promoted on the chat and
+provide links to your reviews and merged Pull Requests.
+Then, you will be able to name yourself a coala developer!
```

Comment:
I think we should toughen the process for a developer. There are too many inactive developers at the moment. Also we could probably add a review of the review they did as a requirement. This saves us from bad merged due to not so focused `acks`